RoslynPluginGenerator/RoslynPluginJarBuilder.cs

```diff
@@ -41,10 +42,10 @@ public class RoslynPluginJarBuilder
         private const string PluginClassName = "org.sonar.plugins.roslynsdk.RoslynSdkGeneratedPlugin";
 
         // Locations in the jar where various file should be embedded
-        private const string RelativeManifestResourcePath = "META-INF\\MANIFEST.MF";
+        private const string RelativeManifestResourcePath = "META-INF/MANIFEST.MF";
 
-        private const string RelativeConfigurationResourcePath = "org\\sonar\\plugins\\roslynsdk\\configuration.xml";
-        private const string RelativeRulesXmlResourcePath = "org\\sonar\\plugins\\roslynsdk\\rules.xml";
+        private const string RelativeConfigurationResourcePath = "org/sonar/plugins/roslynsdk/configuration.xml";
+        private const string RelativeRulesXmlResourcePath = "org/sonar/plugins/roslynsdk/rules.xml";
```

Comment:
The relative path is used for both file paths and in the archive entry names. It has to be / for the archive entries, but can be either for the file paths so it's simpler just to to use /.

Reply:
Maybe that's worth having this feedback directly in the code so we don't change it on the future (forgetting why this was done). Besides, maybe having a test on this (changing from private to internal) might help.

Reply:
I'll add a comment but I don't think it's worth adding further tests. The only one where it really matters is for the ManifestResource path, and there are about a dozen tests that already fail if that is wrong. The other strings are passed to the _ArchiveUpdater.AddFile_ method, which is defensive and replaces the backslashes with forward slashes anyway.